packages/commonwealth/client/scripts/views/components/share_popover.tsx

```diff
@@ -84,8 +84,10 @@ export const SharePopover = ({
                 '_blank',
               );
             } else if (commentId) {
+              const currentRouteSansCommentParam =
```

Comment:
```diff
- currentRouteSansCommentParam
+ currentRouteHasCommentParam
```

Reply:
this actually should be `sans` as it is without comment param